ui/cypress/e2e/collectors.test.ts

```diff
@@ -21,30 +21,41 @@ describe('Collectors', () => {
       const newConfig = 'New Config'
       const configDescription = 'This is a new config testing'
 
-      cy.getByTestID('table-row').should('have.length', 0)
-
-      cy.contains('Create Configuration').click()
-      cy.getByTestID('overlay--container').within(() => {
-        cy.getByTestID('telegraf-plugins--System').click()
-        cy.getByTestID('next').click()
-        cy.getByInputName('name')
-          .clear()
-          .type(newConfig)
-        cy.getByInputName('description')
-          .clear()
-          .type(configDescription)
-        cy.get('.cf-button')
-          .contains('Create and Verify')
-          .click()
-        cy.getByTestID('streaming').within(() => {
-          cy.get('.cf-button')
-            .contains('Listen for Data')
+      cy.getByTestID('table-row')
+        .should('have.length', 0)
+        .then(() => {
+          cy.contains('Create Configuration')
             .click()
+            .then(() => {
+              cy.getByTestID('overlay--container').then(() => {
```

Comment:
`.within` achieves the same sequential behavior as a `then` but also guarantees the rest of our selectors will be within said container.  
```suggestion
              cy.getByTestID('overlay--container').within(() => {
```

Reply:
mistakenly removed this one, didn't go back through the undo stack far enough i think heh